### PR TITLE
Hotfix dd master error step

### PR DESCRIPTION
**Pull request by binggaofw**
# Description
update max retry to 2 
remove greeting part for callback bot 
integrate callback and unblock bot

## Checklist
- [ ] Update CHANGELOG.md
- [ x] Unit tests have been added/updated
- [ ] e2e tests have been added/updated

### CODE REVIEW

**Review by gurunadhambuddepu — APPROVED** [review]

File: src/dialogs/callbackDialogs/getPreferredMethodOfContactStep.ts [hunk 1]
```diff
@@ -159,8 +159,8 @@ export class GetPreferredMethodOfContactStep extends ComponentDialog {
         );
 
       // Proceed with Text Message
-      case "promptConfirmChoiceText":
-        console.log("INTENT: ", intent);
+      case 'promptConfirmChoiceText':
+        console.log('INTENT: ', intent);
         callbackBotDetails.getPreferredMethodOfContactStep = true;
         callbackBotDetails.preferredText = true;
         // callbackBotDetails.confirmPhoneStep = null;
```
Comment (rajuchetlapalli) [inline src/dialogs/callbackDialogs/getPreferredMethodOfContactStep.ts:166]: You can get rid of it. If you don't need it.

Comment (rajuchetlapalli) [inline src/dialogs/callbackDialogs/getPreferredMethodOfContactStep.ts]: You can delete it If you don't need.

File: src/dialogs/callbackDialogs/getPreferredMethodOfContactStep.ts [hunk 2]
```diff
@@ -172,23 +172,23 @@ export class GetPreferredMethodOfContactStep extends ComponentDialog {
       // return await stepContext.endDialog(callbackBotDetails);
 
       // Proceed with Both Messages
-      case "promptConfirmChoiceBoth":
-        console.log("INTENT: ", intent);
+      case 'promptConfirmChoiceBoth':
+        console.log('INTENT: ', intent);
         callbackBotDetails.getPreferredMethodOfContactStep = true;
         callbackBotDetails.preferredEmailAndText = true;
         // await stepContext.context.sendActivity(sendBothMsg);
```
Comment (rajuchetlapalli) [inline src/dialogs/callbackDialogs/getPreferredMethodOfContactStep.ts]: same here

File: src/dialogs/callbackDialogs/getUserEmailStep.ts [hunk 3]
```diff
@@ -43,28 +43,28 @@ export class GetUserEmailStep extends ComponentDialog {
...
 
     // Check if the error count is greater than the max threshold
     if (callbackBotDetails.errorCount.getUserEmailStep >= MAX_ERROR_COUNT) {
       // Throw the master error flag
       // callbackBotDetails.masterError = true;
-      const errorMsg = i18n.__("emailFormatMaxErrorMsg");
+      const errorMsg = i18n.__('emailFormatMaxErrorMsg');
 
       // Send master error message
       // await stepContext.context.sendActivity(errorMsg);
```
Comment (rajuchetlapalli) [inline src/dialogs/callbackDialogs/getUserEmailStep.ts]: same here

File: src/dialogs/callbackDialogs/getUserPhoneNumberStep.ts [hunk 4]
```diff
@@ -149,33 +149,33 @@ export class GetUserPhoneNumberStep extends ComponentDialog {
...
       default: {
         // Catch all
-        console.log("NONE INTENT");
+        console.log('NONE INTENT');
         // Result has come through
         const results = stepContext.result;
         if (results) {
           // phone number validation
           const re =
             /^[\+]?[(]?[0-9]{3}[)]?[-\s\.]?[0-9]{3}[-\s\.]?[0-9]{4,6}$/im;
```
Comment (rajuchetlapalli) [inline src/dialogs/callbackDialogs/getUserPhoneNumberStep.ts]: As per MH comments we are accepting only Canadian phone numbers for Proto bot.

File: src/dialogs/unblockDialogs/unblockLookup.ts [hunk 5]
```diff
@@ -143,18 +143,18 @@ export class ConfirmLookIntoStep extends ComponentDialog {
     );
 
     // Top intent tell us which cognitive service to use.
-    //const intent = LuisRecognizer.topIntent(recognizerResult, 'None', 0.5);
+    // const intent = LuisRecognizer.topIntent(recognizerResult, 'None', 0.5);
 
     // const recognizer = LUISUnblockSetup(stepContext);
     // const recognizerResult = await recognizer.recognize(stepContext.context);
```
Comment (rajuchetlapalli) [inline src/dialogs/unblockDialogs/unblockLookup.ts]: same here

**Review by rajuchetlapalli — CHANGES_REQUESTED** [review]

**Review by llakewood — APPROVED** [review]

**Review by llakewood — APPROVED** [review]

**Comment by llakewood** [comment]
I see that a lot of the Lint App code checks are failing due to changes of single and double quotes. Is there a reason that we are using the opposite quotes to the linting?

**Comment by binggaofw** [comment]
> I see that a lot of the Lint App code checks are failing due to changes of single and double quotes. Is there a reason that we are using the opposite quotes to the linting?

I will change all double quotes to single one

**Pull request merged**